package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Assists in the creation of ATNA audit trail messages for IHE profiles.",
   "main": "index.js",
   "scripts": {
-    "test": "tap --cov test.js"
+    "pretest": "npm run gen:rng",
+    "test": "tap --cov test.js",
+    "gen:rng": "sed 's/#.*//' dicom.rnc > temp.rnc && trang temp.rnc dicom.rng && rm temp.rnc"
```

Comment:
Ok cool - yeah if the comments are useful then it's good to keep them. But what is the issue exactly with having them in the rnc? Does the trang compiler just not like them?